Inferences/Induction.hpp

```diff
@@ -43,7 +43,7 @@ class TermReplacement : public TermTransformer {
 
 public:
   TermReplacement(Term* o, TermList r, bool replaceSkolems = false, unsigned* var = nullptr)
-    : _o(o), _r(r), _v(var), _replaceSkolems(replaceSkolems)
+    : _o(o), _r(r), _replaceSkolems(replaceSkolems), _v(var), _tv()
```

Comment:
`_tv` will be default-constructed regardless, but if you prefer to be explicit that's cool.

Reply:
Ah, I just realized the order here doesn't matter (and therefore things default constructed are not necessarily listed) and warnings about the order here are just to notify the programmer of possible misordered dependencies among members. Thanks for pointing this out. It's never too late to get rid of a bad habit.